README.md

```diff
@@ -172,7 +174,113 @@ $ docker run -it --network priv-net alpine /bin/sh
     inet6 fe80::42:aeff:feaa:e41c/64 scope link
        valid_lft forever preferred_lft forever
 / #
+```
+### Using Plugin in Swarm mode
 
+Currently the plugin supports the MACVLAN network driver in the swarm mode.
+The plugin needs to be installed in all the swarm nodes.
```

Comment:
I suggest saying something like: "Before performing the following steps, the plugin needs to be installed....."